backend/src/defaultAvailability/entities/defaultAvailability.entity.ts

```diff
@@ -0,0 +1,56 @@
+import { Time } from '@osk/shared';
+import {
+  Column,
+  Entity,
+  FindOperator,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { Instructor } from '../../instructors/entities/instructor.entity';
+
+/**
+ * @see https://github.com/typeorm/typeorm/issues/2390
+ */
+const transformFromTimeToDatabaseType = (value: Time | FindOperator<any>) => {
+  if (value instanceof FindOperator) {
+    const oldOperator = value as unknown as { _value: Time };
+    const newOperator = value as unknown as { _value: string };
+
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
Shall we fix the eslint rule if we want to keep the code "clean" from comments?

Reply:
Well, the property with an underscore is an internal property of typeorm which we can't really change. That's why I have had to disable that rule for this line